cmd/nri-prometheus/main.go

```diff
@@ -20,6 +20,11 @@ func main() {
 			logrus.WithError(err).Fatal("error occurred while running scraper")
 		}
 	} else {
-		logrus.Info("New behaviour!")
+		// todo create a proper emitter that add metrics to an integration and prints them to stdout once scraping and processing is performed
+		cfg.Emitters = []string{"integrationSDK4"}
```

Comment:
i don't think this is a good way to do it. I think we should still use the configuration file to add the emitters.
I guess the idea is to force to use only one (the infra sdk one) but I don't think it's really required. The standalone function and the configuration file will be managed by the Agent so I would leave this to the configuration

Reply:
Sure, this is just a stub waiting for an other piece of work that will connect scraper and emitter in the proper way.
Moreover emitter type is configurable but only one option makes sense when standalone=false.